### PR TITLE
Revamp jwplatform-php

### PR DESCRIPTION
**Pull request by ksindi**
Address https://github.com/jwplayer/jwplatform-php/issues/3 and https://github.com/jwplayer/jwplatform-php/pull/1.

Show working examples.

### CODE REVIEW

File: examples/update_video.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 # Replace xxxx and yyyy with your API key/secret (can be found in the CMS).
```
Comment (boyntoni) [inline examples/update_video.php]: Remove comment

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+JW Platform API Client
```
Comment (boyntoni) [inline README.md]: May be worth making `make run` explicit in the `README`

Comment (ksindi) [reply README.md]: Going to leave it out for now

**Comment by ksindi** [comment]
Thanks @boyntoni!

**Pull request merged**